.github/workflows/main.yml

```diff
@@ -9,10 +9,10 @@ jobs:
     steps:
       - uses: actions/checkout@master
 
-      - name: Use Node.js 16
+      - name: Use Node.js 18
         uses: actions/setup-node@master
         with:
-          node-version: 16
+          node-version: 18
```

Comment:
@PowerKiKi @dotansimha I dropped in all actions Node 16 since it's EOL and nextra requires Node.js at least 18